cupy/_core/core.pyx

```diff
@@ -2356,13 +2356,14 @@ cpdef void warn_on_unsupported_std(tuple options):
                     'that causes RTC to break when the standard is set '
                     'to c++11. Please use c++14, or use ROCm > 7.0'
                 )
+        # TODO(leofang): Update this. We need C++17 now.
```

Comment:
Changes look all straight-forward and good to me, nice!

The comment seems like temporary note?
```suggestion
```